index.d.ts

```diff
@@ -256,19 +256,15 @@ declare module 'gdax' {
     interface WebsocketClientOptions {
         channels?: string[];
     }
+    
 
     export class WebsocketClient {
         constructor(
             productIds: string[],
             websocketURI?: string,
             auth?: WebsocketAuthentication,
             { channels }?: WebsocketClientOptions );
-
-        on(event: 'message', eventHandler: (data:object) => void): void;
```

Comment:
Not sure I understand this change

Reply:
Honestly, I'm not sure I do now either. Pushing an amendment now.